shared/operation.go

```diff
@@ -0,0 +1,5 @@
+package shared
+
+// probably just a serialized string
+type Operation struct {
+}
```

Comment:
what does this represent? just an operation enum + data?

Reply:
Element in a log of operations. Basically what has been executed/synced/unsynced as defined by Curp. Left room for potential more struct members but a serialized string probably works fine.